10.0/apache/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update && apt-get install -y \
 
 # https://docs.nextcloud.com/server/9/admin_manual/installation/source_installation.html
 RUN docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
-  && docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu \
+  && docker-php-ext-configure ldap --with-libdir=lib/*-linux-gnu \
```

Comment:
Rather than the `*-linux-gnu`, you may want to use something similar to [httpd](https://github.com/docker-library/httpd/blob/d5f06d182e52822b281e9c5f992309a281a756a7/2.4/Dockerfile#L91) or [php](https://github.com/docker-library/php/blob/3ee1cd8287f6fc7269a731ceb6199ec8dee8b727/Dockerfile-debian.template#L98-L99). (bonus is that `dpkg-dev` is already installed in the Debian-based php images that you are based on).

It looks like you would need to use the `debMultiarch="$(dpkg-architecture --query DEB_BUILD_MULTIARCH)"` line from [php](https://github.com/docker-library/php/blob/3ee1cd8287f6fc7269a731ceb6199ec8dee8b727/Dockerfile-debian.template#L122).